Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.yml

```diff
@@ -732,8 +731,8 @@ script:
       - 'equal'
       - 'not equal'
     - description: 'When specified, the results returned in the list are limited to vulnerabilities with the specified Vulnerability Priority Rating (VPR), score or scores according to the score operator (vprScoreOperator) argument. VPR values range are 0.1-10.0'
-      name: vprScoreValue
       isArray: true
+      name: vprScoreValue
```

Comment:
Please undo the change here, moving the `name` field below `isArray`.
This just makes you appear on `git blame` as if you've added the field, and for no reason (it's a meaningless change)